lib/ivis_opengl/pielighting.h

```diff
@@ -0,0 +1,103 @@
+/*
+	This file is part of Warzone 2100.
+	Copyright (C) 1999-2004  Eidos Interactive
+	Copyright (C) 2005-2020  Warzone 2100 Project
+
+	Warzone 2100 is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 2 of the License, or
+	(at your option) any later version.
+
+	Warzone 2100 is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with Warzone 2100; if not, write to the Free Software
+	Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+
+#pragma once
+
+#include "lib/ivis_opengl/pietypes.h"
+#include "gfx_api.h"
+#include <glm/glm.hpp>
+#include <memory>
+#include <array>
+
+struct LIGHT
+{
+	Vector3i position = Vector3i(0, 0, 0);
+	UDWORD range;
+	PIELIGHT colour;
+};
+
+
+struct LightMap
```

Comment:
This was part of the MAP_TILE struct before, I extracted this part so that it's easier to determine what happens on the lightmap (and surprisingly, not a lot)